src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Microsoft.VisualStudio.ProjectSystem.Managed.VS.csproj

```diff
@@ -25,7 +25,7 @@
     <Folder Include="Properties\" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Update="ProjectSystem\VS\PropertyPages\DebugPageControl.xaml.cs">
+    <Compile Include="ProjectSystem\VS\PropertyPages\DebugPageControl.xaml.cs">
```

Comment:
Update. We shouldn't need any Includes because they are all being included default (unless the cs file is outside the project cone).